src/elements/common/error-boundary/DefaultError.scss

```diff
@@ -3,3 +3,7 @@
         margin: 8px;
     }
 }
+
+.bcs-is-wider .be .be-default-error {
```

Comment:
`bcs-` is specific to ContentSidebar, maybe move this to ContentSidebar stylesheet?

also are you sure this selector works? usually `be` is a class on the root element which wouldn't be a descendent of `bcs-is-wider`

Reply:
+1 to that - this selector will not target anything as `bcs-is-wider` is being added to the same element as the one with `be` class.

Should this be
```
.be {
    .be-default-error {
        margin: 8px;
    }

    &.bcs-is-wider {
        .be-default-error {
         ...
    }
  }
}
```